_data/sites.json

```
@@ -5696,6 +5696,18 @@
         ]
     },
 
+    {
+        "name": "Draxlr",
+        "url": "https://www.draxlr.com/contact/",
+        "difficulty": "hard",
+        "notes": "Contact customer support to request account deletion. They should respond and will delete any personal information, database credentials and result caches that you have provided along with your account within 4 working days.",
+        "domains": [
+            "draxlr.com",
+            "www.draxlr.com",
```

```suggestion
            "draxlr.com"
```
No need for the www one